### PR TITLE
Use new flag syntax

### PR DESCRIPTION
**Pull request by RockRoller01**
Addresses most must-do points of #7138 

- replaces all old-style flag references with the new syntax
- removes reference lists at the bottom of articles
- one part of the conversion involved stripping all multi newlines, which found a couple newspost with wrong formatting, which also got fixed
- remove flag images as new syntax uses osu-web .svg ones
- remove flag-refrences as its no longer needed.
- edited ASC en + de, outdated others that needed outdating
- fix some incorrect beatmap links the CI found

## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by Walavouchey** [comment]
ci checks should probably be done alongside this or at least sooner than later

**Comment by peppy** [comment]
Can you include the script you used to perform this change please? I'd want to run locally to compare.

**Comment by RockRoller01** [comment]
@peppy you can find it here: https://gitlab.com/RockRoller/osu-wikiflagreplacer

just a small java program to iterate over the entire folder and afterwards fixed all the weird references the CI discovered

File: news/2019-06-03-spring-2019-fanart-contest-results.md [hunk 1]
```diff
@@ -111,5 +111,3 @@ We'd like to offer a huge thank you to all the artists who chose to participate
 From all of us here at the osu!team, we hope you enjoy(ed) your Spring this year, and look forward to the next! We'll be back with another seasonal Fan Art contest later on in the year.
 
 —Ephemeral
-
```
Comment (peppy) [inline news/2019-06-03-spring-2019-fanart-contest-results.md:114]: I get these are valid changes, but might be the wrong PR to apply them in?

Comment (peppy) [reply news/2019-06-03-spring-2019-fanart-contest-results.md:114]: The issue is, I am not reviewing 100k lines of change. The only way I can merge this PR is if I can 100% reproduce the changes locally with zero diff.

**Comment by cl8n** [comment]
#7681 was opened for the same thing and it was already in a state where I could review it, so I cleaned up that one instead. if you're still planning to split this PR, check the diff from TPGPL's branch to this one, because almost all of it should cancel out if both are correct

**Comment by RockRoller01** [comment]
Oh well, fine with me to go with the other one, less work for me.

**Pull request closed without merging**